website/content/docs/sync/github.mdx

```diff
@@ -5,3 +5,82 @@ description: The GitHub destination syncs secrets from Vault to GitHub.
 ---
 
 # GitHub
+
+The GitHub repository sync destination allows Vault to safely synchronize secrets as GitHub repository secrets.
```

Comment:
```suggestion
The GitHub repository sync destination allows Vault to safely synchronize secrets as GitHub repository action secrets.
```
Is it worth this precision so it is clear we do not support codespaces & dependabot secrets?